alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java

```diff
@@ -76,17 +76,21 @@ public AlluxioInterpreter(Properties property) {
     alluxioMasterPort = property.getProperty(ALLUXIO_MASTER_PORT);
   }
 
+  private Stream<String> filteredProperties(String prefix) {
+    return properties.stringPropertyNames().stream().filter(propertyKey -> propertyKey.startsWith(prefix));
```

Comment:
I see the following error in CI.
```
[INFO] --- maven-checkstyle-plugin:2.17:check (checkstyle-fail-build) @ zeppelin-alluxio ---
[INFO] There is 1 error reported by Checkstyle 6.11.2 with zeppelin/checkstyle.xml ruleset.
Error:  src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:[80] (sizes) LineLength: Line is longer than 100 characters (found 107).
```

Reply:
Ahh ! Let me raise a Quick PR for this tomorrow.

Reply:
@Reamer @jongyoul 

Please review the following PR:
https://github.com/apache/zeppelin/pull/4534